backend/dataall/modules/worksheets/services/worksheet_service.py

```diff
@@ -122,3 +129,55 @@ def run_sql_query(session, uri, worksheetUri, sqlQuery):
         )
 
         return AthenaClient.convert_query_output(cursor)
+
+    @staticmethod
+    @ResourcePolicyService.has_resource_permission(RUN_ATHENA_QUERY)
+    @ResourceThresholdRepository.check_invocation_count('nlq')
+    def run_nlq(session, username, uri, prompt, worksheetUri, db_name, table_names):
+        return 'TEST'
```

Comment:
I think we need to un-comment this block of code

Reply:
yup - my final step was testing resource thresholds and didn't want to keep invoking the model 😄 - will uncomment